### DISCUSSION TITLE
媒體識讀能力

### DISCUSSION THREAD

**Discussion by Yanxxx7979** [1101-GS4719-C/Python-Community-GS4719-C#1902] [General]
https://youtu.be/heNHCS9mbNg?si=_6uvhiF0JO4RBO1e
不算百齡果的粉絲 但看到有趣的來賓會想要進去看
看完整部影片 印象最深的其實是 來賓表示在自己做了民調深入現實之後
開始懷疑自己幾十年來所接受的資訊究竟有甚麼是真的 甚麼是假的
近些年大家對於媒體識讀能力以及自我價值觀的建成
想問大家對於媒體識讀及自我價值觀的看法

**Comment by YANRONG0123** [comment]
我覺得現代人基本上都要具備媒體識讀能力，原因是資訊量太過龐大以及繁雜，常常聽信於某一件事情，反而忘記去找尋這件事情的對立面，進而導致判斷錯誤，造成無法挽回的後果，並且下次繼續犯錯。

**Comment by poiu6fi** [comment]
我真的覺得媒體識讀很重要，能幫助我們辨識真偽、抵制虛假信息，避免被誤導，提升個人判斷力和批判能力，尤其現今很多網軍在帶風向，沒有媒體識讀能力的人很容易被洗腦

**Comment by lilishh** [comment]
關於媒體識讀，我認為台灣目前的新聞產業都是以點閱率為主，標題多是煽動的，很容易誤導人民。可以多接受不同管道的消息，來協助自己培養媒體識讀的能力。

**Comment by hunteryang0108** [comment]
台灣現在已經將媒體識讀融入基本教材中從小教起了，我相信未來十幾年後，人們對於資訊的辨認會更加小心。我認為個人價值觀的產生應該要有一套自己預先定義好的標準，才不會太容易受外界資訊的干擾，導致價值觀偏差。

**Comment by peresonacifer** [comment]
媒體識讀能力需要的是多方的查證與確認，不能急著相信單一個甚至多個新聞報導的內容，舉個實例，之前花蓮大地震發生後，我班上同學有人發現新北停班停課的消息，消息在某幾家的新聞報導上，所以很多人在班群上傳，這時班上同學就有認人覺得可能是假新聞，但有人已經開始信我一波了，並已經期待班導進來宣布，結果沒事發生，真的是假消息，此時就有人說把訊息截圖檢舉有錢拿，然後很多人都紛紛收回訊息，場面非常搞笑。

**Comment by Jerryguo888** [comment]
在社交媒體上，媒體識讀能力是非常重要的，像之前網紅曾經批評過某些媒體造假或是斷章取義，在沒有查證的情況下，使用一些具有爭議性的言語來做文章吸引讀者，因此當我們看到驚人或爭議性內容時，可以利用專業的事實查證平台來確認資訊的真實性。
